include/sdf/JointAxis.hh

```diff
@@ -93,6 +94,7 @@ namespace sdf
 
     /// \brief Set the x,y,z components of the axis unit vector.
     /// \param[in] _xyz The x,y,z components of the axis unit vector.
+    /// \throws sdf::AssertionInternalError if the norm of the xyz vector is 0.
```

Comment:
I would prefer returning `Errors` from this function rather than throwing an exception

Reply:
Changed to returning `sdf::Errors` with a `[[nodiscard]]` attribute in https://github.com/osrf/sdformat/pull/312/commits/6ccb301fda9659681f1af1eba97b04dcff70e3e2